src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,17 @@
+module Crocodealer.Config
+       ( Repo(..)
+       , Config(..)
+       ) where
+
+import Crocodealer.Core.Label
+
+newtype Repo = Repo
+    { unRepo :: Text
+    }
+
+data Config = Config
+    { username            :: Maybe Text
+    , repository          :: Maybe Text
```

Comment:
All fields in this data type should be strict. Also, they should be prefixed with the data type name. Like `configUsername` and `configRepository`